acquisitions/src/main/resources/thunderjet/cross-modules/features/check-encumbrances-after-issuing-credit-for-paid-order.feature

```diff
@@ -0,0 +1,84 @@
+  # https://folio-org.atlassian.net/browse/MODFISTO-512
+  Feature: Check the encumbrances after issuing credit when the order is fully paid
+
+    Background:
+      * print karate.info.scenarioName
+      * url baseUrl
+
+      * callonce loginAdmin testAdmin
+      * def okapitokenAdmin = okapitoken
+      * callonce loginRegularUser testUser
+      * def okapitokenUser = okapitoken
+
+      * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+      * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+      * callonce variables
+      * def fundId = callonce uuid1
+      * def budgetId = callonce uuid2
+      * def orderId = callonce uuid3
+      * def poLineId = callonce uuid4
+      * def invoiceId1 = callonce uuid5
+      * def invoiceId2 = callonce uuid6
+      * def invoiceLineId1 = callonce uuid7
+      * def invoiceLineId2 = callonce uuid8
+
+
+    Scenario: Check the encumbrances after issuing credit when the order is fully paid
```

Comment:
This and other response code 200-204 scenarios can be annotated with a @Positive indicator